app/code/Magento/Catalog/Model/ResourceModel/AbstractResource.php

```diff
@@ -568,6 +568,6 @@ public function getAttributeRawValue($entityId, $attribute, $store)
             $attributesData = $_data[1];
         }
 
-        return $attributesData ? $attributesData : false;
+        return $attributesData === false ? false: $attributesData;
```

Comment:
Thanks! Just one more space in `false:` left to add :)

It would be really nice if you squash all changes into one commit and force push into the same branch, but I do not insist on this.